src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -234,7 +242,12 @@ export class InputTimePicker
   @Watch("effectiveLocale")
   effectiveLocaleWatcher(): void {
     this.setInputValue(
-      localizeTimeString(this.value, this.effectiveLocale, this.shouldIncludeSeconds())
+      localizeTimeString(
+        this.value,
+        this.effectiveLocale,
+        this.numberingSystem,
+        this.shouldIncludeSeconds()
+      )
```

Comment:
Wonder if it would be appropriate to turn the arguments here into key/value pairs as an object for readability?

Reply:
I'll probably leave it as is for now. We will probably end up refactoring and removing the function in the long run to cache the formatter; same as I just did when creating `NumberStringFormat`. So no reason to clean it up twice.

Reply:
Sounds fine to me :)